### ISSUE TITLE
[parcel] @parcel/core: Failed to resolve 'solid-js/web' from './src/index.ts'

### ISSUE THREAD

**Issue by yw662** [solidjs/solid#1185]
### Describe the bug

parcel is not woking with:
`import { render } from 'solid-js/web'`
like:

```
 Build failed.

@parcel/core: Failed to resolve 'solid-js/web' from './src/index.ts'

 ()/src/index.ts:1:24
  > 1 | import { render } from 'solid-js/web'
```

.babelrc:
```
{
  "presets": [
    "solid"
  ]
}
```

package.json:
```
  "dependencies": {
    "solid-js": "^1.5.1"
  },
  "devDependencies": {
    "@babel/core": "^7.18.13",
    "@parcel/transformer-sass": "^2.7.0",
    "@types/node": "^18.7.13",
    "@typescript-eslint/eslint-plugin": "^5.35.1",
    "@typescript-eslint/parser": "^5.35.1",
    "babel-preset-solid": "^1.5.1",
    "eslint": "^8.22.0",
    "husky": "^8.0.1",
    "parcel": "^2.7.0",
    "typescript": "^4.8.2"
  }
```

parcel 2.7.0, exactly the same version here, works well with solid.js 1.4.8.

### Your Example Website or App

https://github.com/yw662/solid-151-parcel-270-ts/tree/main

### Steps to Reproduce the Bug or Issue

1. with a project using parcel and solid.js, 
2. upgrade solid.js into 1.5.1,
3. make sure it imports 'solid-js/web',
4. build it and you will see.

### Expected behavior

It should compile.

### Platform

- OS: macos
- Browser: compile time / parcel
- Version: solid 1.5.1, parcel 2.7.0

solid 1.4.8 works.

**Comment by ryansolid** [comment]
I imagine this is related to NodeNext because all we did was start exporting types from the package.json export field and now this.

**Comment by yw662** [comment]
Confirm solved by 1.5.2. Thanks.
